lib/webhook_system/faraday_middleware/encoding.rb

```diff
@@ -0,0 +1,21 @@
+# This is taken from https://github.com/ma2gedev/faraday-encoding,
+# all credits for this goes to Takayuki Matsubara (takayuki.1229+github@gmail.com).
+# We should be able to switch to the original gem when
+# this PR (https://github.com/ma2gedev/faraday-encoding/pull/2) is merged.
+module Faraday
```

Comment:
What happened to the one liner Encoding.find...replicate fix for this instead? this seems a bit .. over involved ..